### PR TITLE
Frontmatter

### PR DESCRIPTION
**Pull request by waqarsaleem**
With these changes, the template should be good to go. Please review.

### CODE REVIEW

File: titlepage.tex [hunk 1]
```diff
@@ -0,0 +1,27 @@
+\begin{titlepage}
+  \begin{center}
+    \vfill
+    \textbf{\Huge \Title}
+    \bigskip
+
+    {\large Kaavish Report\\
+      presented to the academic faculty\\
+      by\\
+      \begin{tabular}{ll}
```
Comment (ma569) [inline titlepage.tex:12]: names of group members seem too close to the text - "presened to the academic faculty by" - and logo. Can we shift  the names and logo down by increasing margin?

File: approval.tex [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+This Kaavish project was supervised by:\\\bigskip\\\bigskip\\\bigskip
+\hfill %
+\begin{tabular}{l}
+  \line(1,0){200}\\
+  My Supervisor \\ % Name of your CS supervisor
+  Faculty of Computer Science\\
+  Habib University
+\end{tabular}\\\bigskip
+
+Approved by the Faculty of Computer Science on \hrulefill.
```
Comment (ma569) [inline approval.tex:26]: Do we need external/co-supervisors here

File: report.tex [hunk 3]
```diff
@@ -9,17 +10,33 @@
...
+\usepackage{titlesec}  % customize chapter title 
 \usepackage{upquote}  % prevent mishandling of single quotes in listings
 
-% --- Customize the appearance of hyperref links using \hypersetup
+% TODO: Customize the appearance of hyperref links using \hypersetup
 % See https://en.wikibooks.org/wiki/LaTeX/Hyperlinks#Customization
 
-\addbibresource{references.bib}  % separate file for references
+% Custom chapter format.
+\titleformat{\chapter}[hang]{\bf\huge}{\thechapter.}{2pc}{}
```
Comment (ma569) [inline report.tex:25]: Shall we include chapter numbers in chapter heading?

Can we change the heading of Implementation/Experiments and Results to "Experiments \& Results". This is to ensure the chapter heading is on one line

**Review by ma569 — CHANGES_REQUESTED** [review]
See inline comments

File: approval.tex (hunk 2 above)
Comment (waqarsaleem) [reply approval.tex:26]: Added optional table.

File: report.tex (hunk 3 above)
Comment (waqarsaleem) [reply report.tex:25]: - Changed chapter title.
- I think chapter numbers assist navigation.

File: titlepage.tex (hunk 1 above)
Comment (waqarsaleem) [reply titlepage.tex:12]: fixed spacing.

**Review by ma569 — APPROVED** [review]

**Pull request merged**